pkg/git/git.go

```diff
@@ -8,17 +8,21 @@ import (
 	"strings"
 
 	"github.com/go-logr/logr"
-	"github.com/weaveworks/profiles/api/v1alpha1"
 	"k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/weaveworks/profiles/api/v1alpha1"
 )
 
+// HTTPClient defines an http client which then can be used to test the
+// handler code.
 //go:generate counterfeiter -o fakes/fake_http_client.go . HTTPClient
```

Comment:
I don't think this is quite right?

Reply:
Isn't it? I think this interface is solely there to be able to mock it out, right?

Reply:
Hmm, maybe I didn't quite wrote this correctly.

Reply:
How about:

`// HTTPClient defines a testable base HTTPClient that can be mocked in tests.`

Reply:
I think the fact its used for testing doesn't really belong in the comment description, its more just 
`// HTTPClient defines an interface for HTTP get requests` or something like that

Reply:
Fair point! :)